features/user_has_to_login_to_see_content.feature

```diff
@@ -0,0 +1,29 @@
+@javascript @no_user
+Feature: User has to login to see content of website
+    As a visitor,
+    In order to see content,
+    I need to be logged in.
+
+    Background:
+        Given the following registered RSC exist
+            | email           | password   |
+            | email@email.com | mypassword |
+        And the user is logged in as "email@email.com"
+        And the following jobs exist in the database
+            | name     |
+            | intern   |
+        And I am on the landing page 
+        And I click on "Logout" 
```

Comment:
What's the idea here? You're `logged in as` and then `logout` in the same background-step? Would you even have to log in, in the first place?

Reply:
I already spotted and we're on it ;-)